### PR TITLE
Document how to run the datacube pipeline with a batch job

### PR DESCRIPTION
**Pull request by weiji14**
## What I am changing

- Create documentation on running the datacube pipeline to generate the 13-band GeoTIFF files
- **Preview** at https://clay-foundation.github.io/model/data_datacube.html
- Screenshot:

image

## How I did it

- Copy and paste the markdown text originally created in #54 from the main README.md file to docs/data_datacube.md
- Added a placeholder section

## How you can test it

- Run `jupyter-book build docs/` locally

## Related Issues

Part of #90

### CODE REVIEW

File: docs/data_datacube.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Creating datacubes
+
+## How to create a datacube
+
+TODO:
+- Describe how the `datacube.py` script works for a single MGRS tile
```
Comment (weiji14) [inline docs/data_datacube.md:6]: @yellowcap, do you want to add a few paragraphs on how to run the `datacube.py` script for a single MGRS tile and date range?

File: .github/workflows/deploy-docs.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ name: deploy-book
 # Only run this when the main branch changes
 on:
   # Uncomment the 'pull_request' line below to manually re-build Jupyter Book
-  # pull_request:
+  pull_request:
```
Comment (weiji14) [inline .github/workflows/deploy-docs.yml:7]: Reminder to comment this out before merging
```suggestion
  # pull_request:
```

File: docs/data_datacube.md (hunk 1 above)
Comment (yellowcap) [reply docs/data_datacube.md:6]: done

**Review by yellowcap — APPROVED** [review]
Reviewed and improved. Added requested section and removed the `on_pullrequest` flag.

**Comment by weiji14** [comment]
Cool, thanks Daniel! We just need to revert the docs CI commit at 6d3ca675e6a912b8e521757f1e07dc7d2d44334b before merging this, so that the Jupyter Book site will be built/updated.

**Pull request merged**